src/main/java/com/github/onsdigital/babbage/highcharts/ChartRenderer.java

```diff
@@ -39,18 +63,14 @@ private ChartRenderer() {
      * Optionally takes a width parameter, width is 700 by default, if width exceeds max, max width will be applied, if it is smaller than min, min width will apply
      */
     public void renderChartConfig(HttpServletRequest request, HttpServletResponse response) throws IOException, ContentReadException {
-        String uri = request.getParameter("uri");
+        String uri = request.getParameter(URI_PARAM);
         if (assertUri(uri, request, response)) {
-            ContentResponse contentResponse = ContentClient.getInstance().getContent(uri);
-            LinkedHashMap<String, Object> additionalData = new LinkedHashMap<>();
-            Integer width = getWidth(request);
-            if (width == null) {
-                width = DEFAULT_CHART_WIDTH;
-            }
-            additionalData.put("width", width);
-            String chartConfig = TemplateService.getInstance().renderChartConfiguration(contentResponse.getDataStream(),
+            ContentResponse contentResponse = contentClient.getContent(uri);
+            Map<String, Object> additionalData = new ChartConfigBuilder().width(getWidth(request)).getMap();
+
+            String chartConfig = templateService.renderChartConfiguration(contentResponse.getDataStream(),
```

Comment:
good spot. Given the grief open streams have caused us recently I am slightly embarrassed I missed this.